gulpfile.js

```diff
@@ -123,7 +129,13 @@ gulp.task('pot', function () {
 
 // Checks JavaScript using JSHint
 gulp.task('jshint', function () {
-    return gulp.src([ 'gulpfile.js', path.join( 'openslides', '*', 'static', '**', '*.js' ) ])
+    return gulp.src([
+            'gulpfile.js',
+            path.join( 'openslides', '*', 'static', '**', '*.js' ),
+            '!' + path.join( 'openslides', 'users', 'static', 'js', 'users', 'site.js' ),
+            '!' + path.join( 'openslides', 'motions', 'static', 'js', 'motions', 'base.js' ),
+            '!' + path.join( 'openslides', 'motions', 'static', 'js', 'motions', 'site.js' ),
```

Comment:
These three files are not fixed yet. Someone has to do this and then remove these lines.